### PR TITLE
calico new images for caching

### PR DESCRIPTION
**Pull request by paulgmiller**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/components.json [hunk 1]
```diff
@@ -202,18 +203,20 @@
       "downloadURL": "mcr.microsoft.com/oss/calico/node:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v3.24.6"
+        "v3.24.6",
```
Comment (djsly) [inline parts/linux/cloud-init/artifacts/components.json:206]: should `calico/node` have a `prefetchOptimazations` section ?

**Review by djsly — APPROVED** [review]

File: parts/linux/cloud-init/artifacts/components.json (hunk 1 above)
Comment (cameronmeissner) [reply parts/linux/cloud-init/artifacts/components.json:206]: we haven't prefetched this thus far, though not sure if we really _should_ be

**Comment by cameronmeissner** [comment]
do we still need this?

File: parts/linux/cloud-init/artifacts/components.json (hunk 1 above)
Comment (paulgmiller) [reply parts/linux/cloud-init/artifacts/components.json:206]: I mean I don't actually want to push calcio so maybe the better thing to do is drop it completely. Also are we going to cache 4 versions when I bring in calcio 3.28?

**Comment by paulgmiller** [comment]
Killing Calico caching.

**Pull request closed without merging**